src/backend/Services/OffersServices.cs

```diff
@@ -0,0 +1,34 @@
+﻿using backend.Models;
+using backend.Repositories;
+using System;
+using System.Collections.Generic;
+
+namespace backend.Services
+{
+    public static class OffersServices
+    {
+        public static void SaveOffer(Offer offer)
+        {
+            OffersRepository.SaveOffer(offer);
+        }
+
+        public static Offer GetStandardOffer()
+        {
+            return OffersRepository.GetStandardOffer();
+        }
+
+        public static Offer GetSpecificOffer(int id)
+        {
+            return OffersRepository.GetSpecificOffer(id);
+        }
+        public static List<Offer> GetAllOffers()
+        {
+            return OffersRepository.GetAllOffers();
+        }
+        public static Offer GenerateRandomOfffer()
```

Comment:
Typo (tripple f), should be `GetAllOffers`
```suggestion
        public static Offer GenerateRandomOffer()
```